src/Vektonn.DataSource/Kafka/KafkaTopicCreator.cs

```diff
@@ -20,18 +20,29 @@ public KafkaTopicCreator(ILog log, KafkaProducerConfig kafkaProducerConfig)
             this.log = log;
             kafkaTopicCreationConfig = kafkaProducerConfig.TopicCreationConfig;
 
-            var adminClientConfig = new AdminClientConfig
-            {
-                BootstrapServers = string.Join(",", kafkaProducerConfig.BootstrapServers),
-            };
+            var adminClientConfig = kafkaTopicCreationConfig.GetAdminClientConfig(kafkaProducerConfig.BootstrapServers);
+
+            adminClient = BuildAdmin(adminClientConfig);
+        }
 
-            var adminClientBuilder = new AdminClientBuilder(adminClientConfig)
+        private IAdminClient BuildAdmin(AdminClientConfig adminClientConfig)
+        {
+            return new AdminClientBuilder(adminClientConfig)
                 .SetErrorHandler(
-                    (_, error) => log.Error($"ConfluentAdminClient error: {error.ToPrettyJson()}"))
+                    (_, error) =>
+                    {
+                        if (error.IsFatal)
+                        {
+                            LogExtensions.Error(log, $"ConfluentAdminClient error: {error.ToPrettyJson()}");
```

Comment:
Can we convert static method invocations from `LogExtensions` to extension method calls? (And do it everywhere in solution)

Reply:
https://github.com/vektonn/vektonn/pull/15/commits/22a404eb9668557f02d6cb9126aa38f07f0eb1ad

Done